dev-tools/mage/crossbuild.go

```diff
@@ -44,7 +44,7 @@ const defaultCrossBuildTarget = "golangCrossBuild"
 var Platforms = BuildPlatforms.Defaults()
 
 // SelectedPackageTypes is the list of package types
-var SelectedPackageTypes []PackageType = []PackageType{TarGz}
```

Comment:
where does this get populated in the default case? without digging into it too much it seems something like
```go
	if packageTypes := os.Getenv("PACKAGES"); len(packageTypes) > 0 {
                SelectedPackageTypes = []PackageType{}
		for _, pkgtype := range strings.Split(packageTypes, ",") {
```
should go down around line 56

Reply:
It's been a long time I've played with it the build scripts, by default the release manager is calling `make snapshot` without any arguments, IIRC somewhere deep just read the `dev-tools` yaml files to identify which package to generate. I am reverting a change introduced in https://github.com/elastic/beats/pull/29890. Looking at the commits from the last good commit and now https://github.com/elastic/beats/compare/53c7c51be99a798a484f87d63e633350ecc44df6...master  and the following error. This is the only commit that could introduce a problem.


```
manager/build/projects/gopath/src/github.com/elastic/beats'
15:31:45 make[1]: Leaving directory '/var/lib/jenkins/workspace/elastic+unified-release+master+snapshot-beats-master/cd/release/release-manager/build/projects/gopath/src/github.com/elastic/beats'
15:31:45 Expected but did not find: '/var/lib/jenkins/workspace/elastic+unified-release+master+snapshot-beats-master/cd/release/release-manager/build/projects/gopath/src/github.com/elastic/beats/build/distributions/auditbeat/auditbeat-8.1.0-SNAPSHOT-windows-x86_64.zip'.

```

Reply:
i'm interested in what the objective of the change was, since it isn't readily apparent from the PR that introduced it. the change is small enough that we can dig into it after unblocking the build chain

Reply:
I think it was to make dev testing faster, but there are arguments available to the build mage task to speed up the process. I believe they are not well documented.